cruise.umple/src/UmpleInternalParser_CodeStateMachine.ump

```diff
@@ -956,6 +1002,19 @@ class UmpleInternalParser
     boolean isFinalState = false;
     String changeType = null;
     
+    if (stateToken.getSubToken("final") != null)
+    {
+    	boolean foundInvalidElement = analyzeFinalState(stateToken, fromState);
+    	if (foundInvalidElement)
+    	{
+    		getParseResult().addErrorMessage(new ErrorMessage(72, 
+    														  fromState.getPosition(), 
```

Comment:
@vahdat-ab yikes! I will fix the spaces. It's weird because the arguments line up the way I would like them to in Eclipse
![no_spacing_error](https://cloud.githubusercontent.com/assets/5403082/22574740/a3a2a3d4-e965-11e6-91f7-5d95d1953bc0.png)